### ISSUE TITLE
contribution guide doesn't explain how to label PRs

### ISSUE THREAD

**Issue by evgeni** [ansible/ansible-lint#2061]
##### Summary

PRs have an "ack" workflow, that labels the PR based on rules (inherited from https://github.com/ansible/devtools/blob/main/.github/release-drafter.yml) but this is not documented in https://github.com/ansible/ansible-lint/blob/main/.github/CONTRIBUTING.rst and thus PRs of new contributors end up in red state without any explanation.

##### Issue Type

- Bug Report

##### Ansible and Ansible Lint details

```console (paste below)
ansible --version

ansible-lint --version

```

- ansible installation method: one of source, pip, OS package
- ansible-lint installation method: one of source, pip, OS package

##### STEPS TO REPRODUCE

```console (paste below)

```

##### Desired Behavior

Possible security bugs should be reported via email to `security@ansible.com`

##### Actual Behavior

Please give some details of what is actually happening.
Include a [minimum complete verifiable example] with:

- playbook
- output of running ansible-lint
- if you're getting a stack trace, output of
  `ansible-playbook --syntax-check playbook`

```paste below

```

[minimum complete verifiable example]: http://stackoverflow.com/help/mcve

**Comment by evgeni** [comment]
If external contributors shouldn't bother about this, this also should be documented. Right now, as a contributor, I post a patch and CI returns "failed" without any really good explanation. Like it says "wrong label" in the GHA output, but this is not actionable for me as a contributor.
